projects/gen-ai.md

```diff
@@ -0,0 +1,172 @@
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following existing conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with the instrumentations of existing GenAI Observability vendors for a set of client
+instrumentation libraries that all vendors can depend upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and LLM frameworks for popular libraries in different languages
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel LLM SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+- Python
+  - maintainers:
+    - @lzchen
+    - TBD
+  - approvers/codeowners:
+    - @drewby
+    - @karthikscale3
+    - @nirga
+    - @xrmx
+  - contributors:
+    - @AmanAgarwal041
+    - @alizenhom
+    - @galkleinman
+    - @gyliu513
+    - @lmolkova
+    - @patcher9
+    - *looking for more contributors*
+- JavaScript
+  - maintainers:
+    - @trentm
+    - TBD
+  - approvers/codeowners:
+    - @drewby
+    - @karthikscale3
+    - @nirga
+    - *looking for existing OTel JS approvers/maintainers for general JS & OTel feedback*
```

Comment:
```suggestion
    - @codefromthecrypt
    - *looking for existing OTel JS approvers/maintainers for general JS & OTel feedback*
```

Reply:
It'd be great to have you!

There are [requirements for approvers/maintainers](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md ) status in the community
which boils down to the history of contributions.

The list in this PR is an initial set and is expected to evolve based on the existing process.

Reply:
It was worth a shot!

Meanwhile, I'll open an issue on the community project because there is a chicken/egg condition which is a special case to new projects looking to acquire new people. For example, it discusses "the codebase" whereas no code exists yet. I'll take that offline.